New changes SQL/XT Voidzone add.sql

```diff
@@ -0,0 +1 @@
+UPDATE creature_template SET ScriptName ='npc_xt_void_zone' WHERE entry = 34001;
```

Comment:
both files contain the same, maintain  sql/updates/world/2016_06_25_00_world_335.sql 
renaming it to  sql/updates/world/2016_07_25_99_world.sql